examples/base_train.py

```diff
@@ -24,22 +24,25 @@
 def main():
     # Get command line arguments
     parser = ParlaiParser()
-    parser.add_argument('-n', '--num-examples', default=10)
+    parser.add_argument('-n', '--num-iters', default=10, type=int)
+    parser.add_argument('-a', '--num-agents', default=1, type=int)
     opt = parser.parse_args()
 
-    agent = Agent(opt)
+    agents = []
+    for _ in range(opt['num_agents']):
+        agents.append(Agent(opt))
```

Comment:
so it makes identical agents or?

Reply:
yes it makes identical agents. Just a simple example to demonstrate that a world can be created with any number of agents getting turns in a round robing fashion.